script/docker-start

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e
+
+source $(dirname $0)/generatecert
+
+exec redoctober -addr=:8080 $@
```

Comment:
well in the case where they want to use the container-ip:container-inner-port and use something else its just a nice to have, but ya idk how common that would be, just depends, not everyone exposes ports through localhost though, esp if they are running docker with iptables=false or something